modules/ROOT/partials/configuration/resize.adoc

```diff
@@ -1,7 +1,7 @@
 [[resize]]
 == `+resize+`
 
-This option gives you the ability to disable the resize handle or set it to resize both horizontal and vertically. The option can be true, false or the string `+'both'+`. `+False+` disables the resize, `+true+` enables vertical resizing only, `+'both'+` makes it possible to resize in both directions horizontal and vertical.
+This option allows the resize handle to be disabled or to set the resize handler to resize both horizontally and vertically. The option can be `+true+`, `+false+` or the string `+'both'+`. `+False+` disables resizing, `+true+` enables vertical resizing only, `+'both'+` makes it possible to resize in both directions horizontal and vertical.
```

Comment:
I think dot points are better for listing multi-values.

Reply:
That might be better, but that's well outside the scope of this task so we're better off leaving that for now (even the pronouns tbh were outside of the scope but they are worse).